Examples/java/src/main/java/software/amazon/cryptography/example/hierarchy/CreateKeyExample.java

```diff
@@ -59,7 +67,7 @@ public static String CreateKey(
     // this makes it easy for Crypto Tools to clean up these Example Branch Keys.
     branchKeyId =
       StringUtils.isBlank(branchKeyId)
-        ? "mpl-java-example-hv2-" + java.util.UUID.randomUUID().toString()
+        ? "mpl-java-example-" + java.util.UUID.randomUUID().toString()
```

Comment:
I'm not a big fan of generating the UUID here. I think it's demonstrating an anti-pattern to customers. But if this unblocks something, it could be fine

Reply:
I only do it if the parameter is empty;
I need the parameter to demonstrate what it does and how to use it.
I am trying to strike the middle ground of "it is clear how to create a branch key"
and "this example is a useful native End-to-End test".